### ISSUE TITLE
Announcement: Happy Hacktoberfest 2023!

### ISSUE THREAD

**Issue by edm00se** [edm00se/awesome-board-games#269]
# Add Your Favorite Board Game

If you're looking to contribute for [Hacktoberfest](https://hacktoberfest.com/) and would love an "easy win", but don't want to make it to a superfluous repository, then please consider this one and contribute by adding a favorite board game to this list. There's no need to create an issue, just make sure it's not already in the list.

## What Is This?

This is [an awesome list](https://github.com/sindresorhus/awesome/blob/master/awesome.md) which lets people show their love for, and recommend, board games. I have some CI tooling set up to 1- check for linting issues and 2- create preview builds so we can view opened PRs with [vercel](https://vercel.com/l) (this is all automatic, you don't need to set anything up).

## Common Issues

Image size and improperly formatted tables are the most common culprits. Please follow the contributing and formatting guides, linked to below; they're quick reads and the formatting guide includes an example. If there's an issue with your PR at first, don't worry! I'll help as best as I can and often make code suggestions during a review. If you have questions, ask below in this issue thread or in your PR.

## How Do I Contribute?

Please read the [contributing guide](https://github.com/edm00se/awesome-board-games/blob/master/contributing.md), it's a quick read, and follow the [formatting guide](https://github.com/edm00se/awesome-board-games/blob/master/formatting.md), again it's short and includes examples.

If you're concerned about contributions counting towards hacktoberfest, not to worry! I'll mark any accepted PR with the appropriate label and the repo here is marked already.

Please tag your PRs with this issue \# if you're contributing during October for [Hacktoberfest](https://hacktoberfest.com/). I'll close this issue after the end of October. For now, happy contributing!

### First of All

Welcome! Also, there has been a pattern with this year's new contributors that I feel I must make a recommendation. If you're new to contributing to this repo, please wait until you've completed a successful PR process **before** creating additional pull requests. This will save on my review time and effort, and on the need to go potentially multiple rounds while we sort things out. No names are named, the open PRs are still welcome, but start small before going big please, it will help us both out.

### Secondly

Please limit yourself to a maximum of 2 open Pull Requests for this repository at a time and 5 overall within October. My time is limited and I can only review so many. This year has seen a number of contributors submit large quantities of PRs. Without some self control by contributors, that will make this repository unsustainable.

Why the limitation? Last year for all of October this repo had a total of 108 PRs. This month so far, not having completed a second full week of the month, we're already well over that number.

### Thirdly

Do not `@` mention me or ask for my review. It will happen, in order, in due time. I’m one me, this is a side project, and the effort involved fits into my free time. If you _really_ want to help speed me up, come on over and help me rake my leaves (this is a joke, do not do this).

### Fourth

No "fixing typos" Pull Requests during Hacktoberfest. In spite of taking a bit of reading and edit work, they're low effort and not the variety of contributions that are being requested. If they were requested and an approved issue was created, this would be a different story, but there have been a couple takers this year, each with some of their own incorrect changes, so it is being ruled as out of bounds.

### Fifth

No un-released games. The entire point of this being an [awesome list](https://github.com/sindresorhus/awesome/blob/main/awesome.md) is that it is a game, implied that you've played, and must be awesome, not just good. The last bit is subjective, so I don't make judgements when people add things like Risk or Monopoly, because that's something some people enjoy more than I do. But as for submissions of Kickstarter campaigns that haven't ended, let alone shipped yet, this is just a waste of time and not fitting with this project's intention.

Thank you all in advance for your cooperation.

**Comment by edm00se** [comment]
It would seem we've accumulated a small mountain of PRs. I will attempt to review the outstanding contribution PRs nightly, in the order they are received.

**Comment by edm00se** [comment]
Something I've learned this year is that back when I first created then opened this project up for Hacktoberfest submissions, while I may have had most people find this repo via the "welcome issue" being tagged, that it's the repository tag most people are finding it by now. Usually open source protocol is to check the contributing doc; so I've updated that with a light version of what I had added to the "notes for first time contributors".

If anyone has a chance to review that section and let me know if it covers enough or if it could be clarified, I'd appreciate the input here. Thanks!
